deploy/spectrum-scale-secret.yaml_template

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
```

Comment:
Can we add in this file also 
  labels:
    product: ibm-storage-enabler-for-containers